src/features/common/Layout/UserPropsContext.tsx

```diff
@@ -13,8 +15,8 @@ export const UserPropsContext = React.createContext({
   token: null,
   isLoading: true,
   isAuthenticated: false,
-  loginWithRedirect: ({}) => {},
-  logoutUser: (value: string | undefined) => {},
+  loginWithRedirect: (redirectUri: string | undefined, ui_locales: string | undefined) => {},
+  logoutUser: (returnUrl: string | undefined) => {},
```

Comment:
I wonder if there's a way to remove the TS warning we receive here as the variables are not initially used in the empty function.

Reply:
Not sure how to solve this.